docs/querying/sql-functions.md

```diff
@@ -1077,11 +1077,42 @@ Calculates the base-10 of the numeric expression.
 
 ## LOOKUP
 
-`LOOKUP(<CHARACTER>, <CHARACTER>[, <CHARACTER>])`
+Look up `expr` in a registered [query-time lookup table](lookups.md) named `lookupName`. Returns `replaceMissingValueWith`, which defaults to `null`, when the `expr` is null or when the lookup does not contain a value for `expr`.
```

Comment:
>does not contain a value

Change "value" to "key" for more specificity

Reply:
switched "value" -> "key"

Reply:
Also switched "value" -> "key" on line 1096 for consistency